README.md

```diff
@@ -1 +1,216 @@
-# rubyMorseCode
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Decode Morse Code <a name="about-project"></a>
+
+**Decode Morse Code** decodes a morse code message into text.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Ruby <a name="tech-stack"></a>
+
+<summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Decodes a morse code into text**
+<!-- - **[Stores weight of the animal]** -->
+<!-- - **[Stores date of birth of the animal]** -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+- Live Demo Link not yet available.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- [ ] It would be better if we complete the `Getting Started` section. Currently, it is incomplete with information. `pre-requisite, setup, install, usage` etc. This makes our README file unprofessional